### PR TITLE
Added Footer component

### PR DESCRIPTION
**Pull request by Daria1911**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Added Footer component 

© 2022 GitHub, Inc.
Terms
Privacy
Security
Status

### CODE REVIEW

File: src/components/footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.footer {
+  width: 100%;
+  background: #ef9364;
```
Comment (nikkigraybeal) [inline src/components/footer/Footer.css:3]: I think black background and white text in .footer class to match the rest of the app would look good.

File: src/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function Footer() {
+  return (
+    <div className="footer">
+      <div>
+        <p> Meet the team: </p>
+      </div>
+
+      <div className="teamMembers">
+        <p>
+          Alejandro Patino, Daria Fetko, Olga Musteata, Elena Demidenko, Alzhan
```
Comment (nikkigraybeal) [inline src/components/footer/Footer.js:13]: each team member should be wrapped in their own p tags so we can add linkedIn and GH icons for each person that will lead to individual linkedIn and GH pages.

If you have time to add those icons, that would be awesome - if not, no problem.

File: src/pages/home/Home.js
Comment (nikkigraybeal) [inline src/pages/home/Home.js:1]: Footer should go in App.js just below closing Routes tag not in Home.js.

**Review by nikkigraybeal — CHANGES_REQUESTED** [review]
Nice work, Dasha! Just a couple of quick changes here and I think it's ready to merge.

File: src/components/footer/Footer.css (hunk 1 above)
Comment (Daria1911) [reply src/components/footer/Footer.css:3]: I followed design: https://www.figma.com/file/F7sNzfbrDwaWdslg5R5ZCS/PRACT-Flashcard-App?type=design&node-id=0-88&mode=design&t=Y5C7OGzA6Jv8GSNO-0

**Review by nikkigraybeal — CHANGES_REQUESTED** [review]
Nice progress, Dasha! The footer needs to be fixed to the bottom of the page. If you look on the Registration page and scroll to the bottom, you will see the problem. 

Also, if there is any way to make the icons very small and appear next to the person's name, I think that would be good. Maybe would be good to consult with Alejandro on the styling. It's tricky to fit all of that content into such a small space. I wonder if we should only show the team members on very large screens and then maybe just a link to the about page on smaller screens? Or just switch to showing a site map instead?

**Pull request merged**